include/extractor/extraction_containers.hpp

```diff
@@ -75,6 +72,8 @@ class ExtractionContainers
     std::vector<InputTrafficSignal> external_traffic_signals;
     TrafficSignals internal_traffic_signals;
 
+    std::vector<NodeBasedEdge> normal_edges;
```

Comment:
Not sure about naming here, tbh haven't tried to dive into the logic there, just made it available from outside.